tests/components/pi_hole/test_init.py

```diff
@@ -0,0 +1,48 @@
+"""Test pi_hole component."""
+
+from homeassistant.components import pi_hole
+
+from asynctest import CoroutineMock
+from hole import Hole
+from tests.common import async_setup_component
+from unittest.mock import patch
```

Comment:
Please sort imports according to PEP8.

https://www.python.org/dev/peps/pep-0008/#imports